client/components/preview.tsx

```diff
@@ -54,11 +85,11 @@ const Preview: NextPage<{ expense: ExpenseModel }> = (props) => {
           <div className="flex flex-row items-center justify-between">
             <div className="flex flex-row items-center">
               <Image src={cryptocat} width={24} height={24} />
-              <p className="ml-2 text-primary text-sm">{expense.user}</p>
+              <p className="ml-2 text-primary text-sm">{truncate(expense.creator, 15)}</p>
             </div>
 
             <button className="bg-btn-gradient text-primary py-2 px-3 rounded-3xl text-sm font-bold">
-              {expense.status}
+              {expense.status ? "Paid" : "Pending"}
```

Comment:
This should actually be the status of the debt rather than the status of the expense.


<img width="712" alt="image" src="https://user-images.githubusercontent.com/49523842/180591018-869a074d-f54c-4fa3-bcde-a932722b4e5b.png">